doc/tutorial/first_steps.md

```diff
@@ -6,6 +6,10 @@ It covers installing and initializing LXD, creating and configuring some instanc
 
 After going through these steps, you will have a general idea of how to use LXD, and you can start exploring more advanced use cases!
 
+   ```{note}
+   Ensure that you have 20MiB free disk space before starting this tutorial.
```

Comment:
```suggestion
   Ensure that you have 20 MiB free disk space before starting this tutorial.
```

Reply:
Also, the full note should not be indented (not sure how this shows up, probably as a blockquote?).

Reply:
That figure came from experimentation - first attempt had only 15 GiB free; so the creation of `ubuntu-vm` failed. I then added 10 GiB space, and all was fine. Then reduced the available space by 5 GiB, all still worked.

Reply:
So that's GiB not MiB?

Reply:
Good catch Tom, I didn't even notice. 👍

Reply:
Thanks for picking that up!